src/main/java/org/assertj/core/api/AbstractObjectAssert.java

```diff
@@ -226,24 +225,54 @@ public SELF hasNoNullFieldsOrProperties() {
     return myself;
   }
 
+  /**
+   * Asserts that the actual object has null fields or properties.
```

Comment:
this is not an accurate description of the assertion

Reply:
What do you exactly mean? 
``` Asserts that the actual object has only null fields or properties. ```
Is it fine? Or did you mean something else?